hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/BrokerOperations.java

```diff
@@ -39,8 +42,11 @@ public BrokerOperations(Map<String, String> kafkaZkConnection, ConfigFactory con
     public BrokerOperations(Map<String, String> kafkaZkConnection, int sessionTimeout, int connectionTimeout, String namespace) {
         zkClients = kafkaZkConnection.entrySet().stream()
                 .collect(toMap(Map.Entry::getKey,
-                               e -> new ZkClient(e.getValue(), sessionTimeout, connectionTimeout, ZKStringSerializer$.MODULE$)));
-
+                               e -> {
+                                   ZkConnection connection = new ZkConnection(e.getValue(), sessionTimeout);
+                                   ZkClient zkClient = new ZkClient(connection, connectionTimeout, ZKStringSerializer$.MODULE$);
+                                   return new ZkUtils(zkClient, connection, false);
```

Comment:
Not yet but we're getting closer. My main concern at the moment are offset queues (which I need to manage differently), default settings for kafka consumer (which we should have after some testing on our environment) and whether offset manipulation works as advertised. Remember that scope of this PR is usage of consumer api from kafka 0.10 and not kafka-security. I'm afraid that supporting kafka-security would require changes in hermes-frontend and hermes-management as well.